stonesoup/sensor/radar/radar.py

```diff
@@ -20,6 +20,7 @@
 from ...types.groundtruth import GroundTruthState
 from ...types.numeric import Probability
 from ...types.state import State, StateVector
+from ...models.clutter import ClutterModel
```

Comment:
Thanks for reminding me, I just moved clutter.py to the models folder, not the measurement folder. After the meeting last week, I changed it so that it is not a subclass of the measurement model.